src/tape/ensemble.py

```diff
@@ -1499,6 +1559,12 @@ def from_dask_dataframe(
         self._load_column_mapper(column_mapper, **kwargs)
         source_frame = SourceFrame.from_dask_dataframe(source_frame, self)
 
+        # Repartition before any sorting
+        if npartitions and npartitions > 1:
+            source_frame = source_frame.repartition(npartitions=npartitions)
+        elif partition_size:
+            source_frame = source_frame.repartition(partition_size=partition_size)
```

Comment:
I think we can just add a partition_size pytest parameter and we can get rid of the code coverage warning here

Reply:
Adding two conftest.py functions (one for from_parquet and one for from_dask_dataframes) as I think all the parameters for these loader functions live there